services/basic_types.proto

```diff
@@ -110,6 +110,7 @@ message AccountID {
          */
         bytes alias = 4;
 
+        bytes evmAddress = 5;
```

Comment:
We've been using "snake-case" now---so `evm_address`, `is_default_address`, etc.

Reply:
Renamed